Source/Controllers/DataImport/SPDataImport.m

```diff
@@ -1935,7 +1935,6 @@ - (void)dealloc
 	if (selectedTableTarget)           SPClear(selectedTableTarget);
 	
 	SPClear(nibObjectsToRelease);
-	SPClear(fileManager);
```

Comment:
Just making sure, was this intentional to remove this SPClear?

Reply:
Intentional. It was being over-released.